Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -139,29 +139,35 @@ NetworkStorageManager::NetworkStorageManager(PAL::SessionID sessionID, Markable<
     : m_sessionID(sessionID)
     , m_queueName(makeString("com.apple.WebKit.Storage.", sessionID.toUInt64(), ".", static_cast<uint64_t>(identifier->data() >> 64), static_cast<uint64_t>(identifier->data())))
     , m_queue(SuspendableWorkQueue::create(m_queueName.utf8().data(), SuspendableWorkQueue::QOS::Default, SuspendableWorkQueue::ShouldLog::Yes))
-    , m_defaultOriginQuota(defaultOriginQuota)
```

Comment:
Why did you make these initializer changes?

Reply:
They are supposed to be accessed from the WorkQueue only

Reply:
I hope that's true we'll get new weird behavior :)